app/models/address.rb

```diff
@@ -3,5 +3,16 @@ class Address < ApplicationRecord
   has_many :orders
   
   validates :city, :state, :country, :pincode, presence: true
-  validates :city, uniqueness: { scope: [:state, :country, :pincode] }
+
+  validates :city, uniqueness: { scope: [:state, :country, :pincode], message: ', State, Country and Pincode combination already taken' }, on: :create
```

Comment:
what will happen in all these validations if I edit an existing address and make it same as some other address?

Reply:
fixed